localstack/services/stepfunctions/asl/antlr/ASLLexer.g4

```diff
@@ -1,205 +1,185 @@
-NUMERICGREATERTHANEQUALSPATH: '"NumericGreaterThanEqualsPath"';
-NUMERICLESSTHAN: '"NumericLessThan"';
-NUMERICLESSTHANPATH: '"NumericLessThanPath"';
-NUMERICLESSTHANEQUALS: '"NumericLessThanEquals"';
-NUMERICLESSTHANEQUALSPATH: '"NumericLessThanEqualsPath"';
-OR: '"Or"';
-STRINGEQUALS: '"StringEquals"';
-STRINGEQUALSPATH: '"StringEqualsPath"';
-STRINGGREATERTHAN: '"StringGreaterThan"';
-STRINGGREATERTHANPATH: '"StringGreaterThanPath"';
-STRINGGREATERTHANEQUALS: '"StringGreaterThanEquals"';
-STRINGGREATERTHANEQUALSPATH: '"StringGreaterThanEqualsPath"';
-STRINGLESSTHAN: '"StringLessThan"';
-STRINGLESSTHANPATH: '"StringLessThanPath"';
-STRINGLESSTHANEQUALS: '"StringLessThanEquals"';
-STRINGLESSTHANEQUALSPATH: '"StringLessThanEqualsPath"';
-STRINGMATCHES: '"StringMatches"';
-TIMESTAMPEQUALS: '"TimestampEquals"';
-TIMESTAMPEQUALSPATH: '"TimestampEqualsPath"';
-TIMESTAMPGREATERTHAN: '"TimestampGreaterThan"';
-TIMESTAMPGREATERTHANPATH: '"TimestampGreaterThanPath"';
-TIMESTAMPGREATERTHANEQUALS: '"TimestampGreaterThanEquals"';
-TIMESTAMPGREATERTHANEQUALSPATH: '"TimestampGreaterThanEqualsPath"';
-TIMESTAMPLESSTHAN: '"TimestampLessThan"';
-TIMESTAMPLESSTHANPATH: '"TimestampLessThanPath"';
-TIMESTAMPLESSTHANEQUALS: '"TimestampLessThanEquals"';
-TIMESTAMPLESSTHANEQUALSPATH: '"TimestampLessThanEqualsPath"';
-
-SECONDSPATH: '"SecondsPath"';
-SECONDS: '"Seconds"';
-TIMESTAMPPATH: '"TimestampPath"';
-TIMESTAMP: '"Timestamp"';
-
-TIMEOUTSECONDS: '"TimeoutSeconds"';
-TIMEOUTSECONDSPATH: '"TimeoutSecondsPath"';
-
-HEARTBEATSECONDS: '"HeartbeatSeconds"';
-HEARTBEATSECONDSPATH: '"HeartbeatSecondsPath"';
-
-PROCESSORCONFIG: '"ProcessorConfig"';
-MODE: '"Mode"';
-INLINE: '"INLINE"';
-DISTRIBUTED: '"DISTRIBUTED"';
-EXECUTIONTYPE: '"ExecutionType"';
-STANDARD: '"STANDARD"';
-
-ITEMPROCESSOR: '"ItemProcessor"';
-ITERATOR: '"Iterator"';
-ITEMSELECTOR: '"ItemSelector"';
-MAXCONCURRENCY: '"MaxConcurrency"';
-
-RESOURCE: '"Resource"';
-INPUTPATH: '"InputPath"';
-OUTPUTPATH: '"OutputPath"';
-ITEMSPATH: '"ItemsPath"';
-RESULTPATH: '"ResultPath"';
-RESULT: '"Result"';
-PARAMETERS: '"Parameters"';
-RESULTSELECTOR: '"ResultSelector"';
-
-ITEMREADER: '"ItemReader"';
-READERCONFIG: '"ReaderConfig"';
-INPUTTYPE: '"InputType"';
-CSVHEADERLOCATION: '"CSVHeaderLocation"';
-CSVHEADERS: '"CSVHeaders"';
-MAXITEMS: '"MaxItems"';
-MAXITEMSPATH: '"MaxItemsPath"';
-
-NEXT: '"Next"';
-END: '"End"';
-
-CAUSE: '"Cause"';
-CAUSEPATH: '"CausePath"';
-ERROR: '"Error"';
-ERRORPATH: '"ErrorPath"';
+COMMENT   : '"Comment"';
+STATES    : '"States"';
+STARTAT   : '"StartAt"';
+NEXTSTATE : '"NextState"';
+VERSION   : '"Version"';
+
+TYPE     : '"Type"';
+TASK     : '"Task"';
+CHOICE   : '"Choice"';
+FAIL     : '"Fail"';
+SUCCEED  : '"Succeed"';
+PASS     : '"Pass"';
+WAIT     : '"Wait"';
+PARALLEL : '"Parallel"';
+MAP      : '"Map"';
+
+CHOICES  : '"Choices"';
+VARIABLE : '"Variable"';
+DEFAULT  : '"Default"';
+BRANCHES : '"Branches"';
+
+AND                            : '"And"';
```

Comment:
just an observation: it's always a trade-off for long blocks: nice readability through indentation but a single line change can update the entire block. Overall, I think the alignment is still a great readability improvement for such grammar specs.

Reply:
I totally agree. For the time being the maximum line width is set to 150, it seems to be standard practice in https://github.com/antlr/grammars-v4